overlord/devicestate/devicemgr.go

```diff
@@ -142,11 +160,6 @@ func (m *DeviceManager) ensureOperational() error {
 		return nil
 	}
 
-	if serialRequestURL == "" {
```

Comment:
@zyga it's a top level var,  this check is a left over from when the feature was wip